tests/gui/TestGui.cpp

```diff
@@ -686,6 +686,21 @@ void TestGui::testSearch()
     QTRY_COMPARE(m_dbWidget->currentMode(), DatabaseWidget::ViewMode);
 }
 
+/**
+ * @author Fonic <https://github.com/fonic>
+ * Update clicks within entry view referencing column indices to account
+ * for changed column indices due to new way of showing/hiding column Entry
+ * Model::ParentGroup. This column now has fixed index 0 wether it's shown
+ * or hidden, thus all indices need to be shifted by +1 when not in search
+ * mode (which is the case within this entire method)
```

Comment:
Previous implementation was broken and has now completely been replaced. No need to keep historic comments about previously existing code. We don't maintain a public API which needs deprecation notes.